chrome/content/test/tests.js

```diff
@@ -48,38 +75,125 @@ describe("First Run Suite", function() {
   
   it("tests opening first run tab", function() {
     expect(loadReason).toBe("install");
-    var flag = false;
-    var len = tabs.length;
-    for (var i = 0; i < len; i++) {
-      if (tabs[i].url === "chrome://privly/content/privly-applications/Pages/ChromeFirstRun.html") {
-        flag = true;
-      }
-    }
-    expect(flag).toBe(true);
+    expect(isPageOpen("chrome://privly/content/privly-applications/Pages/ChromeFirstRun.html", true)).toBe(true);  
```

Comment:
It looks like you are writing the jasmine tests as integration tests instead of unit tests. With an integration test you are testing whether everything is plugging together properly, but with a unit test you are testing whether an individual function is working properly. Testing the individual function often requires isolating it from the integrated components.

Depending on what is accessible to the selenium tests, it may be better to write tests like these with Selenium instead of Jasmine. I would guess that you have made the right decision here, but I want you to separate specs that are meant to be unit tests and specs that are meant to be integration tests.